src/dbnode/generated/proto/namespace/namespace.proto

```diff
@@ -15,6 +15,12 @@ message IndexOptions {
     int64 blockSizeNanos = 2;
 }
 
+message SchemaOption {
+    uint32 version = 1;
+    bytes  definition = 2;
```

Comment:
Why is there a definition and a message?

Reply:
I will change it to name. it tells us which message inside the file descriptor in case there are multiple messages (e.g. nested)